src/sentry/integrations/vercel/webhook.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+import six
+import hmac
+import hashlib
+import logging
+
+from django.utils.crypto import constant_time_compare
+from django.views.decorators.csrf import csrf_exempt
+from sentry import options
+from sentry.api.base import Endpoint
+from sentry.web.decorators import transaction_start
+
+logger = logging.getLogger("sentry.integrations.vercel.webhooks")
+
+
+def verify_signature(request):
+    signature = request.META["HTTP_X_ZEIT_SIGNATURE"]
+    secret = options.get("vercel.client-secret")
+
+    expected = hmac.new(
+        key=secret.encode("utf-8"), msg=six.binary_type(request.body), digestmod=hashlib.sha1
+    ).hexdigest()
+    return constant_time_compare(expected, signature)
+
+
+class VercelWebhookEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @csrf_exempt
+    def dispatch(self, request, *args, **kwargs):
+        return super(VercelWebhookEndpoint, self).dispatch(request, *args, **kwargs)
+
+    @transaction_start("VercelWebhookEndpoint")
```

Comment:
@MeredithAnya Smart idea to use transactions here. I'll have to keep that in mind :)